tests/unittests/DeviceManagerTest.cpp

```diff
@@ -392,6 +392,8 @@ TEST_P(DeviceManagerTest, ReuseModule) {
   EXPECT_NE(ctx1, ctx2);
 }
 
+#ifdef GLOW_WITH_CPU
+
```

Comment:
I think checking backendKind on line 400 and instantiating right device manager is slightly better fix.